src/jdk.management/share/classes/com/sun/management/internal/DiagnosticCommandImpl.java

```diff
@@ -149,13 +149,6 @@ private class Wrapper {
         }
 
         public String execute(String[] args) {
-            if (permission != null) {
```

Comment:
There is a lot of code that remains related to assigning a value to `permission`. It looks like it should also be removed.

Reply:
Right, Permission in the class Wrapper, commandDescriptor manipulates a map with permission info, there's native code that passes this between the JVM and library code....
Would like to do that separately in:
JDK-8334493: Remove SecurityManager Permissions infrastructure from DiagnosticCommands